pkg/controller/user-cluster-controller-manager/resources/resources/mla/prometheus/deployment.go

```diff
@@ -133,6 +137,49 @@ func DeploymentCreator() reconciling.NamedDeploymentCreatorGetter {
 						},
 					},
 				},
+				{
+					Name:            "prometheus-config-reloader",
+					Image:           fmt.Sprintf("%s/%s:%s", resources.RegistryQuay, reloaderImageName, reloaderTag),
+					ImagePullPolicy: corev1.PullAlways,
+					Args: []string{
+						// Fully usage of prometheus-config-reloader:
+						// https://github.com/prometheus-operator/prometheus-operator/blob/v0.49.0/cmd/prometheus-config-reloader/main.go#L72-L108
+						"--listen-address=:8080",
+						"--watch-interval=10s",
```

Comment:
@rastislavs From https://github.com/prometheus-operator/prometheus-operator/blob/v0.49.0/cmd/prometheus-config-reloader/main.go#L79, 
`watch-interval` means "how often the reloader re-reads the configuration file and directories", do you think re-reading/checking configuration every 10s is sufficient?

Reply:
yeah, 10s is good enough IMO